applications/minotari_app_grpc/proto/base_node.proto

```diff
@@ -182,6 +184,21 @@ message NewBlockTemplateRequest{
     uint64 max_weight = 2;
 }
 
+/// request  type of GetNewBlockWithCoinbasesRequest
+message GetNewBlockWithCoinbasesRequest{
```

Comment:
Would it not be better to just have one call instead of getting the Template and then calling this? The miner has no reason to make two calls anymore. 

The miner previously would add a coinbase but now it shouldn't do anything with the `NewBlockTemplate`